tools/mo/openvino/tools/mo/middle/DilatedConvolution.py

```diff
@@ -30,29 +30,37 @@ def pattern(self):
                 ('conv', dict(kind='op', op=lambda value: value in ['Conv2D', 'DepthwiseConv2dNative', 'Conv3D'])),
                 ('space_to_batch', dict(kind='op', op='SpaceToBatch')),
                 ('batch_to_space', dict(kind='op', op='BatchToSpace')),
+                ('stb_pad_begin', dict(kind='op', op='Const')),
+                ('stb_pad_end', dict(kind='op', op='Const')),
+                ('bts_crop_begin', dict(kind='op', op='Const')),
+                ('bts_crop_end', dict(kind='op', op='Const')),
                 ('input', dict(kind='data')),
                 ('output', dict(kind='data')),
                 ('conv_output', dict(kind='data')),
                 ('stb_output', dict(kind='data')),
                 ('stb_bs', dict(kind='data')),
-                ('stb_pad_begin', dict(kind='data')),
-                ('stb_pad_end', dict(kind='data')),
+                ('stb_pad_begin_d', dict(kind='data')),
+                ('stb_pad_end_d', dict(kind='data')),
                 ('bts_bs', dict(kind='data')),
-                ('bts_crop_begin', dict(kind='data')),
-                ('bts_crop_end', dict(kind='data'))
+                ('bts_crop_begin_d', dict(kind='data')),
+                ('bts_crop_end_d', dict(kind='data'))
```

Comment:
don't we impact models that relying on the current pattern? Now you introduce additional constraints.
Can you check one model please? Ping personally which one.

Reply:
I checked the model you sent me. Performance is unaffected.